docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -774,7 +775,7 @@
               register: new_reboot
               when: waiting_after_new_reboot.failed
```

Comment:
Do we need to consider ignore_unreachable in this case?

Reply:
i think not required for now.